knowledge-base/snyk/actions/action-security.yml

```diff
@@ -0,0 +1,5 @@
+name: 'Snyk-Docker action'
```

Comment:
Name should be `Snyk Docker` as per https://github.com/snyk/actions/blob/master/docker/action.yml